### PR TITLE
global called prior to conditiional statement

### PR DESCRIPTION
**Pull request by bdolor**
for reference:  https://github.com/pressbooks/pressbooks/commit/4cfe7534593693320d9a40b4a31171a947f5773c

### CODE REVIEW

**Comment by connerbw** [comment]
What? This will create an undefined variable in `_pb_minimum_php()` so it's not gonna work. (Look at the end of the printf() line in an IDE that underlines errors)

What part of the code I submitted didn't work for you? `pressbooks.php` is already in global space. Declaring global in that file is sort of redundant.

**Comment by bdolor** [comment]
Have you tested PB with PBT on a version with PHP less than 5.6? It fails to load themes. I couldn't get any meaningful error messages from it — sorry. 

With the code left as is, PBT does not work with the latest PB on a machine with PHP < 5.6. 

To recreate:
- have a php version less than 5.6
- have the latest PB (3.0) and the latest PBT 
- override `global $pb_minimum_php` in `wp-config.php` with a value like `5.4.0`

Load an admin page with both PB and PBT activated and the appearance of the site changes from red to black. No PB themes will load. That's all I have at the moment.

File: pressbooks.php [hunk 1]
```diff
@@ -77,9 +77,9 @@ function _pressbooks_autoload( $class_name ) {
 // -------------------------------------------------------------------------------------------------------------------
 
 // Override PHP version at your own risk!
+global $pb_minimum_php;
 if ( ! isset( $pb_minimum_php ) ) $pb_minimum_php = '5.6.0';
 function _pb_minimum_php() {
-	global $pb_minimum_php;
 	echo '<div id="message" class="error fade"><p>';
```
Comment (connerbw) [inline pressbooks.php]: Don't remove this global. Both are needed.

**Comment by connerbw** [comment]
Ok, I did some more testing. Looks like something gets cached somewhere because I can sometimes get this working with global, sometimes without. I can accept the pull request if you modify it to put the other global back. Sorry for the confusion.

**Comment by connerbw** [comment]
One thing I can mention about PBT. Is that if PressBooks fails to load I get:

```
[21-Oct-2015 01:31:33 UTC] PHP Fatal error:  Class 'Pressbooks\Book' not found in /var/www/pressbooks/wp-content/plugins/pressbooks-textbook/admin/class-pbt-textbook-admin.php on line 51
[21-Oct-2015 01:31:33 UTC] PHP Stack trace:
[21-Oct-2015 01:31:33 UTC] PHP   1. {main}() /var/www/pressbooks/wp-admin/index.php:0
[21-Oct-2015 01:31:33 UTC] PHP   2. require_once() /var/www/pressbooks/wp-admin/index.php:10
[21-Oct-2015 01:31:33 UTC] PHP   3. require() /var/www/pressbooks/wp-admin/admin.php:138
[21-Oct-2015 01:31:33 UTC] PHP   4. require_once() /var/www/pressbooks/wp-admin/menu.php:271
[21-Oct-2015 01:31:33 UTC] PHP   5. do_action() /var/www/pressbooks/wp-admin/includes/menu.php:149
[21-Oct-2015 01:31:33 UTC] PHP   6. call_user_func_array:{/var/www/pressbooks/wp-includes/plugin.php:503}() /var/www/pressbooks/wp-includes/plugin.php:503
[21-Oct-2015 01:31:33 UTC] PHP   7. PBT\Admin\TextbookAdmin->adminMenuAdjuster() /var/www/pressbooks/wp-includes/plugin.php:503
```

Pressbooks can be activated, but not working. In this example I changed the version to `9000.0.0` try to break things, and I got a Fatal error.

**Comment by connerbw** [comment]
Thanks!

I think we should pull out that autoloader, PHP version check, and anything else that should be shared into a file to simulate a successful failed upgrade (?!)  A sort of file we can conditionally include once without activating Pressbooks.

Ideas welcome.

**Pull request merged**